staging/src/k8s.io/apimachinery/pkg/util/wait/backoff.go

```diff
@@ -349,8 +349,8 @@ type exponentialBackoffManagerImpl struct {
 //	  Steps:    int(math.Ceil(float64(max) / float64(init))), // now a required argument
 //	  Factor:   factor,
 //	  Jitter:   jitter,
-//	}.DelayWithReset(reset, clock)
-//	wait.BackoffUntil(..., delayFn.Timer(), ...)
+//	}.DelayWithReset(clock, reset)
+//	delayFn.Until(...)
```

Comment:
nit: can we use the actual parameters names in the doc, `init/initBackoff`, `max/maxBackoff`, `reset/resetDuration`, `factor/backoffFactor`

Reply:
Thanks, done.